content/source/docs/enterprise/before-installing/centos-requirements.html.md

```diff
@@ -9,7 +9,7 @@ When installing Terraform Enterprise on CentOS Linux, ensure your OS and Docker
 
 ## Install Requirements
 
-* CentOS Linux version 7.1611+
+* CentOS Linux version 7.1611 - 7.8
```

Comment:
CentOS 7.1611 is CentOS 7.3. The same way CentOS 7.2003 is CentOS 7.8. Since the Replicated documentation states CentOS 7.4 to 7.8 (they are changing it from 7.7 to 7.8) I would have this mirror it.

```suggestion
* CentOS Linux version 7.3 - 7.8
```

Reply:
I like the idea of this matching Replicated documentation to improve consistency of messaging.